crates/starknet/src/starknet/estimations.rs

```diff
@@ -89,7 +89,7 @@ fn estimate_transaction_fee(
     let total_l1_gas_usage = l1_gas_usage as f64 + l1_gas_by_vm_usage;
     let total_l1_gas_usage = total_l1_gas_usage.ceil() as u64;
 
-    let gas_price = block_context.gas_price as u64;
+    let gas_price = block_context.gas_prices.eth_l1_gas_price as u64;
```

Comment:
Surely there is not other gas price to use?

Reply:
there is the strk_l1_gas_price, but we dont depend on it as of this state of the project